Gemfile

```diff
@@ -12,7 +12,18 @@ end
 
 group :test do
   # ActiveSupport required to test compatibility with ActiveSupport Core Extensions.
-  gem 'activesupport', '~> 4.x', require: false
+  if RUBY_VERSION >= '2.2.2'
```

Comment:
Theoretically you're not allowed to compare like this. Use the hashie extension from #375, `RubyVersion.new(RUBY_VERSION) >= RubyVersion.new(...)`.